doc/classes/GeometryInstance.xml

```diff
@@ -14,32 +14,54 @@
 	</methods>
 	<members>
 		<member name="cast_shadow" type="int" setter="set_cast_shadows_setting" getter="get_cast_shadows_setting" enum="GeometryInstance.ShadowCastingSetting">
+			The currently selected shadow casting flag.
 		</member>
 		<member name="extra_cull_margin" type="float" setter="set_extra_cull_margin" getter="get_extra_cull_margin">
+			The extra distance added to the GeometryInstance's bounding box ([AABB]) to increase it's cull box.
 		</member>
 		<member name="lod_max_distance" type="float" setter="set_lod_max_distance" getter="get_lod_max_distance">
+			The max distance for the LOD of this GeometryInstance.
 		</member>
 		<member name="lod_max_hysteresis" type="float" setter="set_lod_max_hysteresis" getter="get_lod_max_hysteresis">
+			The max margin for the LOD of this GeometryInstance.
 		</member>
 		<member name="lod_min_distance" type="float" setter="set_lod_min_distance" getter="get_lod_min_distance">
+			The min distance for the LOD of this GeometryInstance.
 		</member>
 		<member name="lod_min_hysteresis" type="float" setter="set_lod_min_hysteresis" getter="get_lod_min_hysteresis">
+			The min margin for the LOD of this GeometryInstance.
 		</member>
 		<member name="material_override" type="Material" setter="set_material_override" getter="get_material_override">
+			The material override for the whole geometry.
+
```

Comment:
Please don't add empty lines, one line == one paragraph already. Use `git commit --amend` to modify that commit (see [PR workflow](http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html)).